sirepo/package_data/template/controls/parameters.py.jinja

```diff
@@ -154,26 +156,28 @@ def read_cols_and_save_report(default_file, save_file, cols_file):
         os.rename(default_file, save_file)
 
 def update_and_run_simulation(correctors):
-    is_success = run_simulation(correctors)
-    readings = []
-    columns = madx_parser.parse_tfs_file('twiss.file.tfs')
-    for idx in range(len(columns.name)):
-        keyword = columns.keyword[idx].replace('"', '')
-        if keyword == 'MONITOR':
-            readings += [float(columns.x[idx]), float(columns.y[idx])]
-        elif keyword == 'HMONITOR':
-            readings += [float(columns.x[idx])]
-        elif keyword == 'VMONITOR':
-            readings += [float(columns.y[idx])]
-    res = np.array(readings)
-    if is_success:
+    global finished_sims
+    if run_simulation(correctors):
+        readings = []
+        finished_sims.append(1)
+        columns = madx_parser.parse_tfs_file('twiss.file.tfs')
+        for idx in range(len(columns.name)):
```

Comment:
I think this should be columns.keyword, since that's the first thing referenced. Indeed, you might just say:

```py
for i, k in enumerate(columns.keyword):
```

I don't see a lot of value in variable names like `idx` vs `i`. I don't like the culture of removing vowels to short names, in general.